openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -68,6 +68,11 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             -openpower_version_filename $(OPENPOWER_PNOR_VERSION_FILE)
 
         $(INSTALL) $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME) $(BINARIES_DIR)
+
+        if [ "$(BR2_OPENPOWER_PNOR_UPDATE_FILENAME)" != "" ]; then \
```

Comment:
A comment here would be great.  Otherwise it isn't real clear why or what we're doing here.